internal/service/federatedsettingsidentityprovider/data_source_federated_settings_identity_provider.go

```diff
@@ -231,12 +228,19 @@ func DataSource() *schema.Resource {
 				Type:     schema.TypeString,
 				Computed: true,
 			},
+			"description": {
+				Type:     schema.TypeString,
+				Computed: true,
+			},
+			"authorization_type": {
```

Comment:
just double checking but docs and examples will come in a follow up PR?

Reply:
Yes we plan to do examples and doc changes as part of another task once Workloads OIDC is implemented.